src/extensions/nginx-app-protect/monitoring/processor/nap.go

```diff
@@ -515,13 +544,34 @@ func (f *NAPConfig) getViolations(logger *logrus.Entry) []*models.ViolationData
 				isB64Decoded bool
 			)
 			if v.Uri != "" {
+				name = violationContextUri
 				value = v.Uri
 			} else if v.UriObjectData != (UriObjectData{}) {
+				name = violationContextUri
 				value = v.UriObjectData.Object
 			} else if v.UriLength != "" {
 				isB64Decoded = true
 				name = fmt.Sprintf("URI length: %s", v.UriLength)
 				value = fmt.Sprintf("URI length limit: %s", v.UriLengthLimit)
+			} else if v.HeaderData != (Header{}) {
```

Comment:
question: should this be Uri Data or Header Data? I ask this as it seems like we are parsing the URI Context information here.